### DISCUSSION TITLE
Woox R7051 - how can i trigger the alarm in an automation?

### DISCUSSION THREAD

**Discussion by ugidj** [Koenkk/zigbee2mqtt#17285] [Devices]
I don't understand how to trigger/activate the siren with an automation/button. it is not seen as a "siren" by the home assistant and therefore I cannot activate it with the siren.turn_on command. how could I do? thank you so much for your precious help

**Comment by merlinpimpim** [comment]
Have you found the solution? Can you share?

Another question about this device:
When connected to power (Z2M-->Exposes-->"ac-connected"=True), in the Z2M dashboard the îcone is not a power outlet but a question mark.
Why ?

![image](https://github.com/Koenkk/zigbee2mqtt/assets/113300765/d3a56d9e-ad9f-4e09-99f9-bcc8097d3a93)

![image](https://github.com/Koenkk/zigbee2mqtt/assets/113300765/8133d1ad-23e4-4160-b3ae-33d02588e328)

**Comment by JanZach85** [comment]
Use scenes in Zigbee2mqtt - woox device details, Set scenes, reboot HA and reboot zigbee2mqtt (yes, each solo). Then set the automation with this scenes. Easy, but 2 hours of testing.
